ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestWorker.java

```diff
@@ -1008,6 +1003,22 @@ public void testWorkerAndInitiatorVersion() throws Exception {
 
   }
 
+  @Test
+  public void testFindNextCompactThrowsTException() throws Exception {
+    LOG.debug("Starting minorTableWithBase");
+
+    Worker worker = Mockito.spy(new Worker());
+
```

Comment:
Removed as many as possible to keep the Arrange-Act-Assert structure. If I remove more lines, it will be really hard to figure out what the test exactly does.